src/handler.ts

```diff
@@ -223,22 +231,30 @@ export
 class FileHandler extends AbstractFileHandler {
   /**
    * Get file contents given a path.
+   *
+   * #### Notes
+   * Subclasses are free to use any or none of the information in
+   * the model.
```

Comment:
Let's put this comment up on the abstract class too.  That's what I use for the notebook, for example, so that's where I would have looked for the docs like this.